rust/src/measurements/make_private_expr/expr_count/mod.rs

```diff
@@ -0,0 +1,145 @@
+use crate::core::{ExprFunction, Measure, MetricSpace, PrivacyMap};
+use crate::domains::MarginPub;
+use crate::metrics::PartitionDistance;
+use crate::transformations::traits::UnboundedMetric;
+use crate::transformations::StableExpr;
+use crate::{
+    core::{Function, Measurement},
+    domains::ExprDomain,
+    error::Fallible,
+};
+
+use num::Zero;
+use polars::lazy::dsl::Expr;
+use polars_plan::dsl::AggExpr;
+
+/// Make a measurement that computes the count exactly under bounded-DP
+///
+/// | input_metric                              |
+/// | ----------------------------------------- |
+/// | `PartitionDistance<SymmetricDistance>`    |
+/// | `PartitionDistance<InsertDeleteDistance>` |
+///
+/// # Arguments
+/// * `input_domain` - ExprDomain
+/// * `input_metric` - valid selections shown in table above
+/// * `expr` - count expression
+pub fn make_expr_private_count<MI: 'static + UnboundedMetric, MO: 'static + Measure>(
+    input_domain: ExprDomain,
+    input_metric: PartitionDistance<MI>,
+    expr: Expr,
+) -> Fallible<Measurement<ExprDomain, Expr, PartitionDistance<MI>, MO>>
+where
+    MO::Distance: Zero,
+    Expr: StableExpr<PartitionDistance<MI>, PartitionDistance<MI>>,
+    (ExprDomain, PartitionDistance<MI>): MetricSpace,
+{
+    let Expr::Agg(AggExpr::Count(expr, include_nulls)) = expr else {
```

Comment:
I was thinking `include_nulls` was something that needed to be included for compatibility with the polars interface, but I don't see it on their `count()`. We do need it?

Reply:
Many user-facing APIs are serviced by the same underlying LogicalPlan variant or Expr variant. In this case:
* len (true): https://docs.rs/polars-plan/0.39.2/src/polars_plan/dsl/mod.rs.html#984
* count (false): https://docs.rs/polars-plan/0.39.2/src/polars_plan/dsl/mod.rs.html#980